src/ts/components/subtitleoverlay.ts

```diff
@@ -85,6 +85,13 @@ export class SubtitleOverlay extends Container<SubtitleOverlayConfig> {
 
       this.applyConfToDom(labelToAdd.getDomElement());
 
+      if (event.position != null) {
+        let domElement = labelToAdd.getDomElement();
+        domElement.css('position', 'fixed');
+        domElement.css('left', `${event.position.column}em`);
```

Comment:
Not really we discussed it  a bit with @dweinber, comparing with jwplayer and youtube implementation.

It's not really possible to properly align subtitles, because we don't use monospaced fonts.